tensorflow_addons/optimizers/novograd.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Novograd for TensorFlow."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.python.training import training_ops
```

Comment:
Seems that private API is the only way to import this. @seanpmorgan Sean, what do you think about this?

Reply:
Hi @shreyashpatodia, could we add some comments here? Like
```python
# TODO: Find public API alternatives to these
from tensorflow.python.training import training_ops
```

Reply:
Done.